plugins/inputs/t128_metrics/t128_metrics.go

```diff
@@ -0,0 +1,298 @@
+package t128_metrics
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"sort"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+const (
+	// DefaultRequestTimeout is the request timeout if none is configured
+	DefaultRequestTimeout = time.Second * 5
```

Comment:
Might read slightly better as `	DefaultRequestTimeout = 5 * time.Second`. This is old code thought so feel free to ignore

Reply:
Yeah, probably better. I don't think I'll change it now though.